src/os_win32.c

```diff
@@ -7893,3 +7893,29 @@ is_conpty_stable(void)
 {
     return conpty_stable;
 }
+
+#ifndef FEAT_GUI_W32
+
+void redraw_ncwindow(void)
```

Comment:
What does `nc` mean?

Reply:
Non-client like WM_NCPAINT etc...

Reply:
I don't think that this function redraws non-client window.
Just resize the console buffer to fit with the console window, doesn't it?
`resize_console_buf` or something might be better?

Reply:
I have written the purpose as it is. I should have written the behavior.
I'll fix it now.